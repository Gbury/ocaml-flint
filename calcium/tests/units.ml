let ctx = Calcium.CTX.mk ()

let () =
  let f = Calcium.of_int ~ctx 42 in
  Format.printf "f:%a@." (Calcium.pp ~ctx) f

let () =
  let f2 = Calcium.of_z ~ctx (Z.of_int 42) in
  Format.printf "z:%a@." (Calcium.pp ~ctx) f2
