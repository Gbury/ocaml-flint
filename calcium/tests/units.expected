f:42
z:42
